app/models/miq_queue.rb

```diff
@@ -153,6 +153,19 @@ def self.put(options)
     msg
   end
 
+  # Execute a job on all servers.  Filterable by zone
+  #
+  def self.broadcast(options)
+    server_targets = MiqServer.select(:id, :guid).select("zones.name AS zone_name")
```

Comment:
Should this only go to active servers?  I'm not sure

Reply:
👍 for limiting to active servers